packages/elastic-eslint-config-kibana/.eslintrc.js

```diff
@@ -92,5 +92,6 @@ module.exports = {
     ],
 
     '@kbn/eslint/no_async_promise_body': 'error',
+    '@kbn/eslint/no_async_foreach': 'error',
```

Comment:
Yeah, I'll look at a similar solution for that.

Reply:
>  I don't want to overload this rule though and have a special error message for that scenario.

Would you suggest implementing a custom rule for every use-case when `(...args: any[]) => Promise<any>` is used instead of `(...args: any[]) => void`?

Reply:
I do think that having custom error messages for each situation would be nice, though I'm not convinced there are that many scenarios right now. It's possibly a bigger more wide issue than I'm thinking though.